proto/viam/app/v1/app.proto

```diff
@@ -1243,11 +1251,31 @@ message UpdateModuleResponse {
   string url = 1;
 }
 
+message ModuleUpdateMetadata {
+  // A list of models that are available in the module
+  repeated Model models = 1;
+  // The executable to run to start the module program
+  string entrypoint = 3;
+}
+
+message MLModelUpdateMetadata {
+  viam.app.mltraining.v1.ModelType model_type = 1;
+  viam.app.mltraining.v1.ModelFramework model_framework = 2;
+}
+
+message MLTrainingUpdateMetadata {
+  viam.app.mltraining.v1.ModelType model_type = 1;
+  viam.app.mltraining.v1.ModelFramework model_framework = 2;
+  bool draft = 3;
+}
+
 message Model {
   // The colon-delimited-triplet of the api implemented by the model
   string api = 1;
   // The colon-delimited-triplet of the model
   string model = 2;
+  // The markdown content describing the usage of the model
+  optional string markdown_documentation = 3;
```

Comment:
would it be annoying to make a further nested object `ModelMetadata`? i can imagine we might want to include more things here later, and it feels a little jammed in `Model`

Reply:
also, what was the decision to have separate documentation for each model rather than the entire module?

Reply:
I think its fine to have high level fields for now. There will be an attributes field added in the coming months but still will be 1-to-1 to a model. I think when nesting protos, it makes sense when you need a list, not much rationale otherwise.

> documentation for each model rather than the entire module

These are actually two projects that will both be done. Model-level documentation is higher priority because it solves the problem of users configuring machines. Later we will support markdown at the module level so that the module detail page can be more content rich than just the `description` field we have now which only allows unformatted text.

Inline Model Documentation Epic (this PR): https://viam.atlassian.net/browse/APP-6907
Embed Markdown in registry items Epic (future project): https://viam.atlassian.net/browse/APP-2349

Reply:
ohh i see. i didn't understand which project this was for